modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/routes/RejectionHandling.scala

```diff
@@ -31,10 +31,10 @@ object RejectionHandling {
       .handle {
         case MalformedQueryParamRejection(_, _, Some(e: WrongOrInvalidJson)) =>
           complete(BadRequest -> (e: HttpRejection))
-        case MalformedQueryParamRejection(_, _, Some(e: IllegalFilterFormat)) =>
-          complete(BadRequest -> (e: CommonRejections))
-        case MalformedQueryParamRejection(_, _, Some(e: IllegalOutputFormat)) =>
-          complete(BadRequest -> (e: CommonRejections))
+        case MalformedQueryParamRejection(_, _, Some(err)) =>
+          complete(BadRequest -> (IllegalParam(err.getMessage): CommonRejections))
+        case MalformedQueryParamRejection(_, _, _) =>
+          complete(BadRequest -> (IllegalParam(""): CommonRejections))
```

Comment:
empty illegal param?

Reply:
When the MalformedQueryParamRejection does not contain a Throwable information